ui/build.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+set -euv -o pipefail
+
+perform_push="${PERFORM_PUSH-false}"
+
+repository=shepmaster
+
+# Build outside the image so nodejs isn't installed on the main image
```

Comment:
Wow, I've wanted that for about three years now. Didn't realise they finally decided to implement it!